### ISSUE TITLE
get_builder_content function using Backbone.js

### ISSUE THREAD

**Issue by AbanobAAkram** [pojome/elementor#5717]
Hello,
I'm trying to develop a custom widget for Elementor in which I use get_builder_content function, I'm trying to use Backbone.js in _content_template function but I can find any JS function that has the same functionality of get_builder_content.
Thanks,
Abanob

**Comment by bainternet** [comment]
@AbanobAAkram 

There is no equivalent JS function you can use. What are you trying to do?

**Comment by AbanobAAkram** [comment]
@bainternet

I'm trying to build a custom widget that has a 'SELECT2' with a list of the created Elementor Templates and get the content of the template that user select, just like the attached image.
I was able to create the function using render() function but it takes much time for rendering while editing so I was intending to use _content_template to boost the rendering speed in the editor page.

I have attached you an image for more clarification.
![index](https://user-images.githubusercontent.com/32802519/45440461-98e43680-b6bc-11e8-8f4a-b77b5f0179cd.png)

Thanks
